index.html

```diff
@@ -0,0 +1,229 @@
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Contact</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <!-- CAROUSEL -->
+    <div class="container-fluid">
+        <div class="row">
+            <div class="carousel-inner">
+                <div id="front slider" class="carousel slide" data-ride="carousel">
+       
+                    <div class="carousel-inner">
+                      <div class="carousel-item active">
+                        <img class="d-block w-100" src="./starter-code/img/slider-1.jpg" alt="First Slide">
+                        <div class="black-overlay"></div>
+                        <div class="carousel-caption d-none d-md-block">
+                          <h3>Slide One</h3>
+                          <p>Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit</p>
+                          <button type="button" class="btn btn-outline-light">Light</button>
+                        </div>
+                      </div>
+                      <div class="carousel-item">
+                        <img class="d-block w-100" src="./starter-code/img/slider-2.jpg" alt="Second Slide">
+                        <div class="carousel-caption d-none d-md-block">
+                          <h3>Slide Two</h3>
+                          <p>Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit</p>
+                          <button type="button" class="btn btn-outline-light">Light</button>
+                        </div>
+                      </div>
+                      <div class="carousel-item">
+                        <img class="d-block w-100" src="./starter-code/img/slider-3.jpg" alt="Third Slide">
+                        <div class="carousel-caption d-none d-md-block">
+                          <h3>Slide Three</h3>
+                          <p>Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit</p>
+                          <button type="button" class="btn btn-outline-light">Light</button>
+                        </div>
+                      </div>
+                    </div>
+                    <!-- CONTROLS -->
+                    <a href="#slider3" class="carousel-control-prev" data-slide="prev">
+                        <span class="carousel-control-prev-icon"></span>
+                      </a>
+            
+                      <a href="#slider3" class="carousel-control-next" data-slide="next">
+                        <span class="carousel-control-next-icon"></span>
+                      </a>
+                </div>
+            </div>
+    
+        </div>
+        
+    </div>
+
+    <!-- RED SECTION-->
+
+    <section class="container-red">
+        <div class="container">
+            <div class="row">
+              <div class="col-sm">
+                <!-- <svg class="bi" width="32" height="32" >
```

Comment:
To render the icons, you must first load the Font Awesome library ` <link
      rel="stylesheet"
      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.9.0/css/all.css"
    />`.

Then you can show the right icons by using the right CSS class names.